### DISCUSSION TITLE
Changing the logo

### DISCUSSION THREAD

**Discussion by IronT3cH** [telekom-security/tpotce#1648] [Q&A]
Hi, I want to know is it possible to change logo in the dashboard?

I have tried to add new logo in the folder ~/tpotce/docker/nginx/dist/html/assets/img/ and change logo file in the file index.html

I restart the tpot, it still get the original one.

![image](https://github.com/user-attachments/assets/f3ecb2d4-3297-41d4-8348-2de9c3880a4c)
![image](https://github.com/user-attachments/assets/3e4c8d22-db35-4f4f-a26f-56dad3735537)

Please give me advice if it is possible.

Thank you

**Comment by jacobholtz** [comment]
I'd like to know this as well
